zookeeper-server/src/main/java/org/apache/zookeeper/ZKUtil.java

```diff
@@ -200,9 +195,7 @@ public static void visitSubTreeDFS(ZooKeeper zk, final String path, boolean watc
     }
 
     @SuppressWarnings("unchecked")
-    private static void visitSubTreeDFSHelper(ZooKeeper zk, final String path,
-        boolean watch, StringCallback cb)
-            throws KeeperException, InterruptedException {
+    private static void visitSubTreeDFSHelper(ZooKeeper zk, final String path, boolean watch, StringCallback cb) throws KeeperException, InterruptedException {
```

Comment:
very long line again

Reply:
This pattern can be resolve easier. Since chop down method declaration is a fixed pattern.